javascript/javascript2/week2/index2.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
remove duplicate file

Reply:
hi aina thanks for you feedback i have now implemented suggested changes. please have a look if i have done it in right way. thank you